doc/syntax.org

```diff
@@ -0,0 +1,43 @@
+#+TITLE:   Markup Syntax Guide for Pancacke(d)
+#+AUTHOR:  Scott Wang
+#+STARTUP: align indent
+#+OPTIONS: H:5
+
+
+* Overview
```

Comment:
What I strongly recommend and want is the meta type, which would make it possible do define functions which can be used at `embeds` and `models` sections.

For example, we can absolutely implement the `hasMany`, `hasOne`, `Enum` and `Range` by define them as meta types.